torntools/scripts/content/global/ttGlobal.js

```diff
@@ -784,3 +787,17 @@ function addPeopleBoxFilter() {
 		};
 	}
 }
+
+function hideGymHighlight() {
+	if (settings.pages.gym.hide_gym_highlight) {
+		if (mobile) {
+			doc.find("a[href='/gym.php'] svg").setAttribute("fill", "url(#sidebar_svg_gradient_regular_mobile)");
+			doc.find("a[href='/gym.php'] svg").setAttribute("filter", "url(#svg_sidebar_mobile)");
+			doc.find("#nav-gym").classList.remove(Object.values(doc.find("#nav-gym").classList)[1]);
```

Comment:
I don't think `Object.values()` is necessary here. The functionality seems to work without it too in my console tests.

Reply:
Also on line #799

Reply:
Also as you run this line in both `if` and `else`, it'd be more stylish to put it after the if/else block. And lastly, wouldn't it be better to use some kind of a certain class name here? What if they change the class order?

Reply:
Done.